shared/chat/conversations-list/index.desktop.js

```diff
@@ -1,62 +1,130 @@
 // @flow
 import React from 'react'
-import {Box, Text, Avatar, Icon, Usernames} from '../../common-adapters'
+import {Box, Text, MultiAvatar, Icon, Usernames} from '../../common-adapters'
 import {globalStyles, globalColors} from '../../styles'
 import {participantFilter} from '../../constants/chat'
 import {formatTimeForConversationList} from '../../util/timestamp'
 
 import type {Props} from './'
+import type {InboxState} from '../../constants/chat'
 
-const ConversationList = ({inbox, onSelectConversation, selectedConversation, onNewChat, nowOverride}: Props) => (
-  <Box style={{...globalStyles.flexBoxColumn, backgroundColor: globalColors.darkBlue4, width: 240}}>
+const AddNewRow = ({onNewChat}: Props) => (
+  <Box
+    style={{...globalStyles.flexBoxRow, ...globalStyles.clickable, minHeight: 48, justifyContent: 'center', alignItems: 'center', flexShrink: 0}}
+    onClick={() => onNewChat()}>
+    <Icon type='iconfont-new' style={{color: globalColors.blue, marginRight: 9}} />
+    <Text type='BodyBigLink'>New chat</Text>
+  </Box>
+)
+
+const rowBorderColor = (idx: number, lastParticipantIndex: number, hasUnread: boolean, isSelected: boolean) => {
+  if (idx === lastParticipantIndex) {
+    if (lastParticipantIndex === 1) {
+      if (hasUnread) {
+        return globalColors.orange
+      }
+      return isSelected ? globalColors.darkBlue2 : globalColors.darkBlue4
+    }
+    return hasUnread ? globalColors.orange : undefined
+  }
+
+  return undefined
+}
+
+const Row = ({onSelectConversation, selectedConversation, onNewChat, nowOverride, conversation}: Props & {conversation: InboxState}) => {
+  const participants = participantFilter(conversation.get('participants'))
+  const isSelected = selectedConversation === conversation.get('conversationIDKey')
+  const isMuted = conversation.get('muted')
+  // $FlowIssue
```

Comment:
flow doesn't like me passing avatar props into the multi since its a union. dunno what a good fix is